src/wiktextract/extractor/en/example.py

```diff
@@ -118,6 +120,8 @@ def extract_template_ja_usex(
     example_data["literal_meaning"] = clean_node(
         wxr, None, node.template_parameters.get("lit", "")
     )
+    clean_example_empty_data(example_data)
+    return example_data
```

Comment:
`example_data` seems to still be the same object as `parent_data`; I think this should be copied. Same in `extract_template_zh_x`, you're receiving `parent_data`, modifying it and returning it. I *think* this results in the function returning the same object over and over again, so the objects in `examples` are the same object.

Reply:
No need to copy at here. The difference is "zh-x" create two examples data, so it need to copy the example data from its parent list before adding new data. But "ja-x" only create one data, and the parent list data is not added.

I have tested some Japanese page, code runs fine.